src/dashboard/tests/test_transfer.py

```diff
@@ -0,0 +1,56 @@
+import os
+
+from django.core.urlresolvers import reverse
+from django.test import TestCase
+from django.test.client import Client
+
+from components import helpers
+from main.models import DublinCore
+
+
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+
+
+class TestTransferViews(TestCase):
+    fixture_files = ['test_user.json', 'transfer.json']
+    fixtures = [os.path.join(THIS_DIR, 'fixtures', p) for p in fixture_files]
+
+    def setUp(self):
+        self.client = Client()
+        self.client.login(username='test', password='test')
+        helpers.set_setting('dashboard_uuid', 'test-uuid')
+
+    def test_metadata_edit(self):
+        """Test the metadata form of a transfer"""
+        transfer_uuid = '3e1e56ed-923b-4b53-84fe-c5c1c0b0cf8e'
+        url = reverse(
+            'components.transfer.views.transfer_metadata_edit',
+            args=[transfer_uuid],
+        )
+        response = self.client.post(url, {
+            'title': 'My First Transfer',
+            'is_part_of': '1234',
+            'creator': 'The Creator',
+            'subject': 'A Subject',
+            'description': 'Transfer description',
+            'publisher': 'The Publisher',
+            'contributor': 'A contributor',
+            'date': '2019-01-01',
```

Comment:
An aside, but it is interesting the `date` field doesn't validate as a date. Nothing to fix here now, but thanks to your tests I was quickly able to try out a few different strings to see the behavior, nice :+1: